bokehjs/src/less/dropdown.less

```diff
@@ -19,23 +19,27 @@
 
   cursor: pointer;
   font-size: var(--font-size);
-  background-color: #fff;
-  border: 1px solid #ccc;
+  background-color: var(--background-color);
+  border: var(--border);
   border-radius: var(--border-radius);
-  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+  box-shadow: var(--box-shadow-x-symmetric);
 
   & > .bk-divider {
     height: 1px;
     margin: calc(var(--line-height-computed)/2 - 1px) 0;
     overflow: hidden;
-    background-color: #e5e5e5;
+    background-color: var(--divider-color);
   }
 
   & > :not(.bk-divider) {
     padding: var(--padding-vertical) var(--padding-horizontal);
 
     &:hover, &.bk-active {
-      background-color: darken(white, 10%);
+      // @matpapp - I suppose we will need to define highlight colors
+      // rather than use lighten/darken functions because in light
+      // mode we would need to darken but in dark mode we would need
+      // to lighten (I think)
+      background-color: darken(var(--background-color), 10%);
```

Comment:
Also we can mix CSS variables with Less functions:
```css
RuntimeError: Error evaluating function `darken`: Argument cannot be evaluated to a color in /home/runner/work/bokeh/bokeh/bokehjs/src/less/dropdown.less on line 42, column 25:
41       // to lighten (I think)
42       background-color: darken(var(--background-color), 10%);
43     }
```

Reply:
We may be able to use `color-mix()` CSS function to replace `lighten()` and `darken()`, but even then it would be good to define lighter/darker variants somewhere. However, it looks like https://github.com/bokeh/bokeh/discussions/14415 provides enough shades to cover these cases.